### ISSUE TITLE
Page unresponsive error when searching in large ontology_array columnType in forms

### ISSUE THREAD

**Issue by BrendaHijmans** [molgenis/molgenis-emx2#1034]
**What version of EMX2 are you using (see footer)**
8.51.0

**Describe the bug**
It is not possible to search through large ontology_array columnTypes in forms.

**To Reproduce**
1. Go to ['https://emx2.test.molgenis.org'](https://catalogue-acc.molgeniscloud.org/testCohort_update/tables/#/Subcohorts)
2. Edit a Subcohort via forms
3. Scroll down to 'mainMedicalCondition'
4. See that data loading is slow. 
5. Try to search for e.g. 'Cholera'
6. it is not possible to type anything, page becomes unresponsive

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by mswertz** [comment]
server is slow, we really need to be careful here. Server is about 10x slower then kubernetes so we think something is wrongly configured there. So can you check if the server response is issue?

**Comment by connoratrug** [comment]
The component that renders the items ( and filters and searches ) downloads the entire ontology to the browser , this only scales to maybe a few hundred items ( depending on client specs ). To fix this issue the render, filter and search need to be done on the server ( filers and search via db or dedicated index) and rendering should be done lazy.

**Comment by mswertz** [comment]
First please verify that that is the issue here indeed

**Comment by connoratrug** [comment]
does not work for large ontologies

**Comment by mswertz** [comment]
Until we have use case this issue stays closed

**Comment by connoratrug** [comment]
EMX2 ABL : _If many categories then stupidly large list of options are being loaded, 100k items_

**Comment by mswertz** [comment]
Agreed. However, in this case the size of the ontology is unreasonable. Should be reduced. Therefore this is a known limitation that will not yet receive high prio
